application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -47,20 +47,35 @@ Analytics.Code.AnalyticsDashboardClass_startDate=Start date
 Analytics.Code.AnalyticsDashboardClass_startDate.hint=Start time of the date interval used on all statistics of this dashboard.
 
 ## Most Viewed Pages Macro
+analytics.mostViewedPages.header.actions=Actions
 analytics.mostViewedPages.header.bounceRate=Bounce rate
 analytics.mostViewedPages.header.date=Date
-analytics.mostViewedPages.header.entryNbVisits=No. of views that started on this page
```

Comment:
I discussed with Stefana whether I should remove this column, because there will be a dedicated macro for entry pages and we came to the conclusion that it would be redundant to have it here.